templates/crud_rest_api_dartfrog/lib/users.dart

```diff
@@ -0,0 +1,9 @@
+final users = [
+  {"name": 'tobi'},
+  {"name": 'loki'},
+  {"name": 'jane'}
+];
+
+bool userExists(String username) {
```

Comment:
given we will have another example with actual jwt auth, do we need this "mock" implementation?